types/react-native/index.d.ts

```diff
@@ -5435,8 +5435,9 @@ export class TouchableNativeFeedback extends TouchableNativeFeedbackBase {
      *
      * @param color The ripple color
      * @param borderless If the ripple can render outside it's bounds
+     * @param radius The radius of ripple effect
      */
-    static Ripple(color: ColorValue, borderless?: boolean): RippleBackgroundPropType;
+    static Ripple(color: ColorValue, borderless: boolean, radius?: number): RippleBackgroundPropType;
```

Comment:
This is a Flow ‘maybe’ type, which also takes `null`:

```suggestion
    static Ripple(color: ColorValue, borderless: boolean, radius?: number | null): RippleBackgroundPropType;
```

Reply:
Ah, good catch, this time I missed it 😅